.github/workflows/manual-release.yml

```diff
@@ -0,0 +1,112 @@
+name: Manual Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      module:
+        description: 'Module name to release (e.g., scheduled-function)'
+        required: true
+        type: string
+      version:
+        description: 'Version number (e.g., 1.0.0)'
+        required: true
+        type: string
+      release_notes:
+        description: 'Release notes (optional)'
+        required: false
+        type: string
+
+permissions:
+  contents: write
+
+jobs:
+  validate-input:
+    name: Validate Input
+    runs-on: ubuntu-latest
+    outputs:
+      module-exists: ${{ steps.check-module.outputs.exists }}
+      version-valid: ${{ steps.check-version.outputs.valid }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Check if module exists
+        id: check-module
+        run: |
+          if [ -d "terraform/modules/${{ github.event.inputs.module }}" ]; then
+            echo "exists=true" >> $GITHUB_OUTPUT
+            echo "✅ Module ${{ github.event.inputs.module }} exists"
+          else
+            echo "exists=false" >> $GITHUB_OUTPUT
+            echo "❌ Module ${{ github.event.inputs.module }} does not exist"
+            exit 1
+          fi
+
+      - name: Validate version format
+        id: check-version
+        run: |
+          version="${{ github.event.inputs.version }}"
+          if [[ $version =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
```

Comment:
Do you want to verify that the version number is greater than the existing released version?  (Or at least isn't identical to any existing released version?)